components/__tests__/report-button.test.tsx

```diff
@@ -12,40 +12,9 @@ describe("ReportButton", () => {
   const province: Province = provinceBuilder();
   const contact: Contact = province.data[0];
   it("renders correctly", () => {
-    const { container } = render(
-      <ReportButton contact={contact} provinceName={province.name} />,
-    );
+    render(<ReportButton contact={contact} provinceName={province.name} />);
 
-    expect(container.firstChild).toMatchInlineSnapshot(`
-      <div
-        class="flex justify-center mt-4"
-      >
-        <button
-          aria-label="Laporkan kesalahan"
-          class="inline-flex flex-row px-4 py-2 text-sm rounded-md items-center justify-center border border-transparent font-medium focus:outline-none focus:ring-2 focus:ring-offset-2 text-blue-700 bg-blue-100 hover:bg-blue-200 focus:ring-blue-500 disabled:cursor-not-allowed disabled:opacity-75 relative z-10"
-          type="button"
-        >
-          <svg
-            aria-hidden="true"
-            class="-ml-1 mr-2 h-5 w-5"
-            fill="none"
-            stroke="currentColor"
-            viewBox="0 0 24 24"
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <path
-              d="M12 8v4m0 4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
-              stroke-linecap="round"
-              stroke-linejoin="round"
-              stroke-width="2"
-            />
-          </svg>
-          <span>
-            Laporkan kesalahan
-          </span>
-        </button>
-      </div>
-    `);
+    expect(screen.getByText(/laporkan kesalahan/gi)).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByText(/laporkan kesalahan/i)).toBeVisible();
```

Please use `.toBeVisible()` assertion instead to ensure that the element is actually visible.
And also avoid the `global` flag in the regex, as I have never gotten it recommended in any @testing-library examples, including the ones I'm getting  from testing-playground.com.